config/routes.rb

```diff
@@ -28,13 +28,10 @@
     resources :items
   end
 
-  resource :login, except: :show do
-    get 'login'
+  resource :sessions, except: [:show, :edit, :update] do
```

Comment:
I wouldn't use the resource directive here.

well well, I think we got something to complete the story with the locale change:
/profile/password -> password change form
/profile/password  POST -> change it
/profile/locale POST -> change locale

what do you think ?

where to put noaccess ?

Reply:
Can be done in the profilerestyling story